decidim-meetings/app/views/decidim/meetings/meetings/_filters.html.erb

```diff
@@ -12,7 +12,13 @@
     </div>
   </div>
 
-  <%= form.collection_radio_buttons :date, [["upcoming", t(".upcoming")], ["past", t(".past")]], :first, :last, legend_title: t(".date") %>
+  <% unless @meetings_warning %>
```

Comment:
I'd use a more meaningful variable name here: @upcoming_meetings or something like that.